.github/ISSUE_TEMPLATE/collab-cycle-kickoff.md

```diff
@@ -32,6 +32,8 @@ Provide links or documents for the following:
 ## Questions
 Please refer to the guidance in [Getting Started with the Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Getting-started.1760493650.html) for detailed information and examples for the below questions.
 
+*For information and examples of Static vs Dynamic pages, click the following link: [Static vs Dynamic](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/static-and-non-static-page-documentation.md)
```

Comment:
same feedback as I had for the Platform Website page. Please move this below Question #6. There's already a note under #6 that is denoted with 1 asterisk. That note is still valid, so make sure to put 2 asterisks on this. I would put the 2 asterisks next to "static pages" - let me know if you have questions.

Reply:
Updated.  Please review.

Reply:
instead of "click the following link", can you say "review the **Static vs. Dynamic guidance**" with the bold text as the link.